src/variorum/Intel/clocks_features.c

```diff
@@ -289,34 +308,34 @@ void set_p_state(int cpu_freq_mhz, enum ctl_domains_e domain, off_t msr_perf_sta
         perf_storage_temp(&pd, msr_perf_status, msr_perf_ctl, domain);
     }
 
-    switch(domain)
+    switch (domain)
     {
-    case SOCKET:
-        printf("Set frequencies per socket\n");
-        for (i = 0; i < nsockets; i++)
-        {
-            *pd->perf_ctl[i] = cpu_freq_mhz / 100 * 256;
-        }
-        write_batch(PERF_CTRL);
-        break;
-    case CORE:
-        printf("Set frequencies per core\n");
-        for (i = 0; i < nthreads; i++)
-        {
-            *pd->perf_ctl[i] = cpu_freq_mhz / 100 * 256;
-        }
+        case SOCKET:
+            printf("Set frequencies per socket\n");
+            for (i = 0; i < nsockets; i++)
+            {
+                *pd->perf_ctl[i] = cpu_freq_mhz / 100 * 256;
+            }
+            write_batch(PERF_CTRL);
+            break;
+        case CORE:
+            printf("Set frequencies per core\n");
+            for (i = 0; i < nthreads; i++)
+            {
+                *pd->perf_ctl[i] = cpu_freq_mhz / 100 * 256;
+            }
 #if VARIORUM_DEBUG
-        printf("PERF_CTL raw decimal %" PRIu64 "\n", *pd->perf_ctl[9]);
+            printf("PERF_CTL raw decimal %" PRIu64 "\n", *pd->perf_ctl[9]);
 #endif
```

Comment:
What is the correct indentation for code within an `#if` and `#endif` block? I saw a few places where there was no indentation, but I see the code being indented here?

Reply:
This has been indented to accommodate indented switch case statement.